tutorials/workzone-build-1-homepage/workzone-build-1-homepage.md

```diff
@@ -2,33 +2,33 @@
 parser: v2
 auto_validation: true
 time: 20
-tags: [ tutorial>beginner, software-product>sap-work-zone]
-primary_tag: software-product>sap-work-zone
+tags: [ tutorial>beginner, software-product>sap-build-work-zone]
+primary_tag: software-product>sap-build-work-zone
 author name: Lindsay Bert
 author_profile: https://github.com/LindsayBert
 ---
 
-# Design SAP Work Zone's Home Page
+# Design a Home page in SAP Build Work Zone, advanced edition
 <!-- description --> Open the Home page, add content to it, and change the overall layout.
 
 ## Prerequisites
-A subaccount in SAP BTP that includes a subscription to the SAP Work Zone service has already been setup for you.
+A subaccount in SAP BTP that includes a subscription to the SAP Build Work Zone, advanced edition service, has already been setup for you.
```

Comment:
Please use unordered list, even when there is only one line item.
Remove full stop.